src/components/Input/index.js

```diff
@@ -0,0 +1,26 @@
+import { useEffect, useState } from "react";
+import * as S from "./styles";
+
+const Input = ({ label }) => {
+  const [InputValue, SetInputValue] = useState("");
+
+  useEffect(() => {
+    const SetConsole = () => {
+      console.log(InputValue); // teste
+    };
+    SetConsole();
+  }, [InputValue]);
+
+  return (
+    <>
+      <S.Container>
+        <S.Label>{label}</S.Label>
+        <S.InputElement
+          onChange={(event) => SetInputValue(event.target.value)}
+        />
+      </S.Container>
+    </>
+  );
+};
```

Comment:
Dessa forma você não vai conseguir capturar o valor do input quando for utiliza-lo, pois o onChange está sendo tratando dentro do componente

Sugiro que crie props: `onChange`, `value` e `id` e passe elas para o InputElement

O `id` seria importante pra você usar o atributo `for` na label, para que quando o usuário clicar na label ele deixe o input em focus.
exemplo: https://www.w3schools.com/tags/att_label_for.asp

Reply:
Feito.